system-tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/local/BlobTransferIntegrationTest.java

```diff
@@ -64,7 +64,8 @@ public void transferBlob_success() {
 
     @NotNull
     private BlobServiceClient getBlobServiceClient(String keyVaultName) {
-        var credential = new DefaultAzureCredentialBuilder().build();
+        // Not using AzureCliCredentialBuilder because of agent issue https://github.com/orgs/github-community/discussions/20830
```

Comment:
```suggestion
        // Not using DefaultAzureCredentialBuilder because of agent issue https://github.com/orgs/github-community/discussions/20830
```